### PR TITLE
New Package: font-hanazono-20141012

### PR DESCRIPTION
**Pull request by diogoleal**
Correcting the problem of commits. #1741

### CODE REVIEW

File: srcpkgs/font-hanazono/template [hunk 1]
```diff
@@ -0,0 +1,29 @@
...
+create_wrksrc=yes
+
+distfiles="http://jaist.dl.sourceforge.jp/hanazono-font/${_revision}/hanazono-${version}.zip"
+checksum=06a7285dcc5c8a22aa3a1b319f965d9d336b84af61fabd5d9d5ba6fa2d861208
+
+do_install() {
+	vmkdir usr/share/fonts/TTF
+	vinstall HanaMinA.ttf 644  usr/share/fonts/TTF
+	vinstall HanaMinB.ttf 644  usr/share/fonts/TTF
+	vmkdir usr/share/license/hanazono/
```
Comment (beefcurtains) [inline srcpkgs/font-hanazono/template]: This is automatic

File: srcpkgs/font-hanazono/template [hunk 2]
```diff
@@ -0,0 +1,29 @@
...
+homepage="http://fonts.jp/hanazono/"
+license="Hanazono Font License, Open Font License"
+depends="font-util xbps-triggers"
+noarch="yes"
+hostmakedepends="unzip"
+font_dirs="/usr/share/fonts/TTF"
+
+create_wrksrc=yes
+
+distfiles="http://jaist.dl.sourceforge.jp/hanazono-font/${_revision}/hanazono-${version}.zip"
```
Comment (beefcurtains) [inline srcpkgs/font-hanazono/template]: Use $SOURCEFORGE_SITE

**Comment by beefcurtains** [comment]
Would it be okay to just do ``license="OFL-1.1"`` and not care about the other license? You only need 1 license to redistribute, so the other license is either redundant or not applicable (i.e. not copyleft/free).

File: srcpkgs/font-hanazono/template (hunk 2 above)
Comment (diogoleal) [inline srcpkgs/font-hanazono/template]: When I put a variable $SOURCEFORCE_SITE the value is  http://downloads.sourceforge.net/sourceforge/hanazono-font/62072/hanazono-20141012.zip, but is downloaded a html page.

This project is located in SourceForge Japan

**Comment by diogoleal** [comment]
The source is distributed under two licenses, the file LICENSE.txt the file begins:

"The Hanazono font is dual-licensed under the Hanazono Font License and the Open Font License"

You think it's valid to leave only 1?

File: srcpkgs/font-hanazono/template (hunk 1 above)
Comment (diogoleal) [inline srcpkgs/font-hanazono/template]: Ops, I will remove.

File: srcpkgs/font-hanazono/template (hunk 2 above)
Comment (diogoleal) [inline srcpkgs/font-hanazono/template]: a example :  without $SOURCEFORGE_SITE:
(I add "echo $distfiles")

 ~/Workspace/void-packages/ [master*]./xbps-src clean font-hanazono
http://jaist.dl.sourceforge.jp/hanazono-font/62072/hanazono-20141012.zip
http://jaist.dl.sourceforge.jp/hanazono-font/62072/hanazono-20141012.zip

 ~/Workspace/void-packages/ [master*] ./xbps-src pkg font-hanazono
http://jaist.dl.sourceforge.jp/hanazono-font/62072/hanazono-20141012.zip
[*] Updating `http://repo.voidlinux.eu/current/x86_64-repodata' ...
[*] Updating `http://repo.voidlinux.eu/current/nonfree/x86_64-repodata' ...
[*] Updating `http://repo.voidlinux.eu/current/multilib/x86_64-repodata' ...
[*] Updating `http://repo.voidlinux.eu/current/multilib/nonfree/x86_64-repodata' ...
http://jaist.dl.sourceforge.jp/hanazono-font/62072/hanazono-20141012.zip
=> font-hanazono-20141012_1: removing autodeps, please wait...
http://jaist.dl.sourceforge.jp/hanazono-font/62072/hanazono-20141012.zip
=> font-hanazono-20141012_1: building ...
   [host] unzip-6.0_6: found (http://repo.voidlinux.eu/current)
   [runtime] font-util>=0: found (http://repo.voidlinux.eu/current)
   [runtime] xbps-triggers>=0: found (http://repo.voidlinux.eu/current)
=> font-hanazono-20141012_1: installing host dependency 'unzip-6.0_6' ...
http://jaist.dl.sourceforge.jp/hanazono-font/62072/hanazono-20141012.zip
=> font-hanazono-20141012_1: running do-fetch hook: 00-distfiles ...
=> font-hanazono-20141012_1: fetching distfile 'hanazono-20141012.zip'...
looking up jaist.dl.sourceforge.jp
connecting to jaist.dl.sourceforge.jp:80
requesting http://jaist.dl.sourceforge.jp/hanazono-font/62072/hanazono-20141012.zip
hanazono-20141012.zip: [26MB 0%] 12KB/s ETA: 01h49m
hanazono-20141012.zip: [26MB 0%] 60KB/s ETA: 10m56s
hanazono-20141012.zip: [26MB 2%] 267KB/s ETA: 02m03s
hanazono-20141012.zip: [26MB 7%] 641KB/s ETA: 00m47s
hanazono-20141012.zip: [26MB 14%] 907KB/s ETA: 00m29s
hanazono-20141012.zip: [26MB 17%] 795KB/s ETA: 00m28s
hanazono-20141012.zip: [26MB 24%] 1058KB/s ETA: 00m21s
hanazono-20141012.zip: [26MB 25%] 807KB/s ETA: 00m25s
hanazono-20141012.zip: [26MB 25%] 684KB/s ETA: 00m28s

with $SOURCEFORGE_SITE:
 ~/Workspace/void-packages/ [master*]rm hostdir/sources/font-hanazono-20141012 -rf
 ~/Workspace/void-packages/ [master*] ./xbps-src clean font-hanazono               
http://downloads.sourceforge.net/sourceforge/hanazono-font/62072/hanazono-20141012.zip
http://downloads.sourceforge.net/sourceforge/hanazono-font/62072/hanazono-20141012.zip
 ~/Workspace/void-packages/ [master*] ./xbps-src pkg font-hanazono                 
http://downloads.sourceforge.net/sourceforge/hanazono-font/62072/hanazono-20141012.zip
[*] Updating `http://repo.voidlinux.eu/current/x86_64-repodata' ...
[*] Updating `http://repo.voidlinux.eu/current/nonfree/x86_64-repodata' ...
[*] Updating `http://repo.voidlinux.eu/current/multilib/x86_64-repodata' ...
[*] Updating `http://repo.voidlinux.eu/current/multilib/nonfree/x86_64-repodata' ...
http://downloads.sourceforge.net/sourceforge/hanazono-font/62072/hanazono-20141012.zip
=> font-hanazono-20141012_1: removing autodeps, please wait...
http://downloads.sourceforge.net/sourceforge/hanazono-font/62072/hanazono-20141012.zip
=> font-hanazono-20141012_1: building ...
   [host] unzip-6.0_6: found (http://repo.voidlinux.eu/current)
   [runtime] font-util>=0: found (http://repo.voidlinux.eu/current)
   [runtime] xbps-triggers>=0: found (http://repo.voidlinux.eu/current)
=> font-hanazono-20141012_1: installing host dependency 'unzip-6.0_6' ...
http://downloads.sourceforge.net/sourceforge/hanazono-font/62072/hanazono-20141012.zip
=> font-hanazono-20141012_1: running do-fetch hook: 00-distfiles ...
=> font-hanazono-20141012_1: fetching distfile 'hanazono-20141012.zip'...
looking up downloads.sourceforge.net
connecting to downloads.sourceforge.net:80
requesting http://downloads.sourceforge.net/sourceforge/hanazono-font/62072/hanazono-20141012.zip
302 redirect to http://sourceforge.net/downloads
looking up sourceforge.net
connecting to sourceforge.net:80
requesting http://sourceforge.net/downloads
301 redirect to http://sourceforge.net/publish
looking up sourceforge.net
connecting to sourceforge.net:80
requesting http://sourceforge.net/publish
301 redirect to http://sourceforge.net/publish/
looking up sourceforge.net
connecting to sourceforge.net:80
requesting http://sourceforge.net/publish/
hanazono-20141012.zip: [17KB 24%] 17KB/s ETA: 00m00s
hanazono-20141012.zip: 17KB [avg rate: 70KB/s]
=> font-hanazono-20141012_1: verifying checksum for distfile 'hanazono-20141012.zip'... 
=> ERROR: SHA256 mismatch for 'hanazono-20141012.zip:'
bc9c15df70ef0c9f762e8a6c56b4ee3167e0a01a686713dbecf90595aa1d9c13
=> ERROR: font-hanazono-20141012_1: couldn't verify distfiles, exiting...

Comment (beefcurtains) [inline srcpkgs/font-hanazono/template]: Oh, my bad. Didn't know there's a Japanese fork. Try this: ``http://dl.osdn.jp/hanazono-font/${_revision}/hanazono-${version}``

**Comment by beefcurtains** [comment]
I think so, but I can't read Japanese.

**Comment by xtraeme** [comment]
Please change the license to `OFL-1.1` and you don't need to install the license file.

**Comment by diogoleal** [comment]
I made the changes you requested.

**Pull request merged**